lib/Runtime/Executor/ThreadPoolExecutor.cpp

```diff
@@ -178,9 +176,8 @@ bool ExecutionState::incrementNodeParentsDone(const DAGNode *node,
   // been created in the constructor.
   auto it = nodeParentsDone_.find(node);
 
-  if (it == nodeParentsDone_.end()) {
-    assert(!"Node parents done counter should exist but not found!");
-  }
+  DCHECK(it != nodeParentsDone_.end())
```

Comment:
`DCHECK_NE()`?

Reply:
In order to do DCHECK_NE, the operands need to have operator== and also operator<< and hashmap iterator doesn't have this I guess. But also, I don't think _NE is that useful here, since the main thing we need to know is that it's not at end() not what the value of the iterator is.